services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/events.py

```diff
@@ -178,7 +205,7 @@ async def action(cls, app: FastAPI, scheduler_data: SchedulerData) -> None:
             ] = await dynamic_sidecar_client.containers_inspect(
                 dynamic_sidecar_endpoint
             )
-        except (httpx.HTTPError, DynamicSidecarNetworkError):
+        except (httpx.HTTPError, DynamicSidecarUnexpectedResponseStatus):
             # After the service creation it takes a bit of time for the container to start
             # If the same message appears in the log multiple times in a row (for the same
             # service) something might be wrong with the service.
```

Comment:
question, you do not log the message as you did before?

Reply:
No, nothing changed here, message is still logged.